emacs-travis.mk

```diff
@@ -42,6 +39,17 @@ MAJOR_VERSION := $(word 1,$(subst ., ,$(EMACS_VERSION)))
 # Whether the version is a release candidate
 PRETEST ?= $(findstring rc,$(PRE_RELEASE_PART))
 
+# Build a minimal Emacs with no special flags, to build as fast as possible
+ifndef EMACSCONFFLAGS
+EMACSCONFFLAGS := --with-x-toolkit=no --without-x --without-all --with-xml2 \
+	CFLAGS='-O2 -march=native' \
+	CXXFLAGS='-O2 -march=native'
+
+ifeq ($(shell test ( $(EMACS_VERSION) -eq snapshot ) -o ( $(MAJOR_VERSION) -ge 25 ); echo $$?),0)
```

Comment:
So you are calling `test` and checking the status code against 0, is that it?

I suppose there's no better way?

Reply:
You could take the approach taken in d00f0b7481c8e929be218d885a2281415f44bfc8 to update GnuTLS on 26.1 and snapshot, but I prefer this approach. While it's a little more verbose, there's no duplication of logic.